### PR TITLE
B-21597 Add Temporary Duty (TDY) Order Type MAIN

### PR DESCRIPTION
**Pull request by traskowskycaci**
## [INT PR](https://github.com/transcom/mymove/pull/14049)

[Extra commit in main](43c50b6de80c4e440fde006f3276efeffda8f7d3) was to resolve a single to double quotes linter issue in swagger/internal.yaml, now matches integration.
Before:
image

After:
image

### CODE REVIEW

File: swagger/internal.yaml [hunk 1]
```diff
@@ -2539,6 +2540,7 @@ definitions:
       WOUNDED_WARRIOR: Wounded Warrior
       BLUEBARK: BLUEBARK
       SAFETY: Safety
+      TEMPORARY_DUTY: Temporary Duty (TDY)
```
Comment (danieljordan-caci) [inline swagger/internal.yaml:2543]: Looks like the only difference is the formatting change here in the yaml

Screenshot 2024-11-07 at 2 21 43 PM

**Review by danieljordan-caci — APPROVED** [review]
Only difference is a lil formatting fun in swagger - LGTM

File: swagger/internal.yaml (hunk 1 above)
Comment (traskowskycaci) [reply swagger/internal.yaml:2543]: Should be fixed with 43c50b6de80c4e440fde006f3276efeffda8f7d3

**Review by cameroncaci — APPROVED** [review]
Commits are good

**Pull request merged**